src/pages/Thanks/ThanksPage.vue

```diff
@@ -288,9 +291,19 @@ export default {
 			.map(item => item.loan);
 
 		// MARS-194-User metrics A/B Optimizely experiment
-		const depositTotal = parseFloat(this.receipt?.totals?.depositTotals?.depositTotal);
-		userHasLentBefore(this.loans.length > 0);
-		userHasDepositBefore(depositTotal > 0);
+		const depositTotal = this.receipt?.totals?.depositTotals?.depositTotal;
+
+		const hasLentBefore = this.loans.length > 0;
+		const hasDepositBefore = parseFloat(depositTotal) > 0;
+
+		this.cookieStore.remove(hasLentBeforeCookie);
+		this.cookieStore.remove(hasDepositBeforeCookie);
```

Comment:
In this two components, why does the cookie need to be removed first? To update the expiration date?

Reply:
Removed!